lib/tasks/db.js

```diff
@@ -37,13 +37,63 @@ module.exports = {
     }
   },
 
+  'db:seed': {
+    descriptions: {
+      'short': 'Run seeders.',
+      'long': [
+        'The command runs every existing seed file.'
+      ]
+    },
+
+    task: function () {
+      getMigrator('seeder', function (migrator) {
+        migrator.pending()
+        .then(function (seeders) {
+          if (seeders.length === 0) {
+            console.log('No seeders were found.');
+            process.exit(0);
+          }
+        }).then(function () {
+          return migrator.up();
+        }).then(function () {
+          process.exit(0);
+        });
+      });
+    }
+  },
+
+  'db:unseed': {
+    descriptions: {
+      'short': 'Deletes data from the database.',
+      'long': [
+        'The command unseeds every existing seed.'
+      ]
+    },
+
+    task: function () {
+      getMigrator('seeder', function (migrator) {
+        migrator.executed()
+        .then(function (seeders) {
+          if (seeders.length === 0) {
+            console.log('No executed seeders found.');
+            process.exit(0);
+          }
+        }).then(function () {
+          return migrator.down({to: 0});
```

Comment:
Is there a specific reason for this to undo all the seeding migrations at once? shouldn't it behave as `db:migrate:undo` or should we add `db:seed:undo`?